src/Microsoft.Fx.Portability/Analysis/AnalysisEngine.cs

```diff
@@ -221,6 +224,74 @@ public IEnumerable<string> FindUnreferencedAssemblies(IEnumerable<string> unrefe
             }
         }
 
+        public IList<NuGetPackageInfo> GetNuGetPackagesInfo(IEnumerable<string> assemblies, IEnumerable<FrameworkName> targets)
+        {
+            var nugetPackages = new List<NuGetPackageInfo>();
```

Comment:
Depending on how this method is used and how often it's called, it might be better to have it use `yield return` so it can stream the package info to wherever it needs

Reply:
Fixed.